tests/ops/test_qubit_ops.py

```diff
@@ -2010,3 +2083,77 @@ def circuit_pauli_x():
         pauli_x_state = circuit_pauli_x()
 
         assert np.allclose(mpmct_state, pauli_x_state)
+
+
+class TestArithmetic:
+    @pytest.mark.parametrize(
+        "wires,input_string,output_string",
+        [
+            ([0, 1, 2, 3], "0000", "0000"),
+            ([0, 1, 2, 3], "0010", "0010"),
+            ([0, 1, 2, 3], "0100", "0110"),
+            ([0, 1, 2, 3], "0110", "0101"),
+            ([0, 1, 2, 3], "1010", "1011"),
```

Comment:
Going over all 16 eventualities + changing to the tape mode with and without expansion should be a comprehensive enough set of tests (and same for Sum)

Reply:
> I like these! Ideally it would be good to cover all 16 eventualities, and confirm by hand that the second bitstring is indeed the carry.

Also that the sum is the correct one, right?